corehq/apps/style/templates/style/includes/analytics_google.html

```diff
@@ -99,7 +99,7 @@
         // rather than the 3rd or 4th from the left, because it's variable
         // https://developers.google.com/analytics/devguides/collection/analyticsjs/events#overview
         var lastArg = arguments[arguments.length - 1];
-        var callBack = lastArg.hitCallback;
+        var callBack = lastArg && lastArg.hitCallback;
```

Comment:
The last argument could be null or undefined, which would cause this line to throw an error.